lib/6to5/transformation/transformers/es6-let-scoping.js

```diff
@@ -359,13 +358,15 @@ LetScoping.prototype.getParams = function (params) {
  */
 
 LetScoping.prototype.getLetReferences = function () {
-  var closurify = false;
-  var self = this;
+  var state = {
+    self: this,
+    closurify: false
+  };
 
   // traverse through this block, stopping on functions and checking if they
   // contain any outside let references
   traverse(this.block, {
-    enter: function (node, parent, scope) {
+    enter: function (node, parent, scope, context, state) {
       if (t.isFunction(node)) {
         traverse(node, {
           enter: function (node, parent) {
```

Comment:
Whoops, I shoulda pass `state` here. It's still being closed over.
If we extract these to outer level and put them separately this mistake will be hard to make.